### DISCUSSION TITLE
Log outline

### DISCUSSION THREAD

**Discussion by t1tan** [variar/klogg#500] [Ideas]
Have a list of parent/child rules (regex/plaintext) that will show-up into a tree control.
The rules might look like this:
    (root): "Start task"
    "Start task": "New process", "Crash encountered", "Start reporting"
    "New process": "Test registered", "Test success", "Test fail"

The end result would be something like this:
```
/-
  |-Start task-
  |                  |-New process-
  |                  |                        |-Test registered
  |                  |                        |-Test success
  |                  |                        |-Test registered
  |                  |                        |-Test fail
  |                  |-Start reporting
  |-Start task-
                     |-New process-
                     |                        |-Test registered
                     |                        |-Test success
                     |-Crash encountered
                     |-Start reporting
```
You would then be able to click on any of the tree nodes and be taken straight to the corresponding line in the file

**Comment by variar** [comment]
That is an interesting suggestion. The ui for this is very different from what is currently in klogg, so I'm not sure if I'll have time to look into this.
